server.go

```diff
@@ -12,8 +12,14 @@ import (
 	"time"
 )
 
-// Maximum number of TCP queries before we close the socket.
-const maxTCPQueries = 128
+const (
+	// Maximum number of TCP queries before we close the socket.
+	maxTCPQueries = 128
+	// Maximum number of incoming DNS messages in queue.
+	maxQueueSize = 1000000
```

Comment:
Why 1000000?

Reply:
CoreDNS cannot handle 1M requests - so this queue size is not affected anything (prevents bottleneck).